<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IhmRondes</class>
 <widget class="QWidget" name="IhmRondes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Liste des pointeaux :</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetPointeaux">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Désignation</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Lieu</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidgetRondes">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>701</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Position</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Désignation</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Lieu</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Temps min</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Temps max</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Images.qrc">
      <normaloff>:/img/images/basIcon.png</normaloff>:/img/images/basIcon.png</iconset>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Déroulement de la ronde :</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Ronde :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxNom"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Appliquer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>390</y>
     <width>193</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Enregistrer</string>
      </property>
      <property name="icon">
       <iconset resource="Images.qrc">
        <normaloff>:/img/images/saveIcon.png</normaloff>:/img/images/saveIcon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Supprimer</string>
      </property>
      <property name="icon">
       <iconset resource="Images.qrc">
        <normaloff>:/img/images/supprIcon.png</normaloff>:/img/images/supprIcon.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>210</y>
     <width>123</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Ajouter le pointeau :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Images.qrc">
        <normaloff>:/img/images/droiteIcon.png</normaloff>:/img/images/droiteIcon.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
